### DISCUSSION TITLE
Need to change clangd defaults

### DISCUSSION THREAD

**Discussion by 56phil** [VonHeikemen/lsp-zero.nvim#124] [Q&A]
Hello. I need help with configuring clangd for my needs. I currently have two issues: 1) I want to change the standard to c++20. 2) I'm using the Boost headers and the LSP is not aware of the location. I've placed it in my $PATH, but that doesn't help. 

Please don't be too hard on me. I'm just an old, retired IT worker desperately trying to get to the 21st century. I'm very new to Lua. Thank you for your help.

**Comment by VonHeikemen** [comment]
So I found this in the neovim forum: [How to integrate the “boost” library into the clangd LSP server?](https://neovim.discourse.group/t/how-to-integrate-the-boost-library-into-the-clangd-lsp-server/3581).

They suggest creating a file called `compile_flags.txt`. I think if you create it in the root of your project then `clangd` should detect it. Anyway, in this file you add the build flags you need for the project. Something like this.

```
-std=c++20
-I/some/path/to/boost/include
-L/some/path/to//boost/lib
```

I'm not a C++ developer so I can't test it but I hope it helps.

**Reply by 56phil** [reply]
That was perfect. Thank you!
